modules/wildfly/src/main/modules/ogm/cassandra-drivers/module.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Hibernate OGM, Domain model persistence for NoSQL datastores
+ ~
+ ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+<module xmlns="urn:jboss:module:1.1" name="org.hibernate.ogm.cassandra.drivers" slot="${hibernate.ogm.cassandra.module.slot}">
```

Comment:
Generally I think we should always do this, as it improves isolation.
For example this way we don't force the user to have an outdated version of Guava.

I already had to apply a similar patch to Neo4J to separate the older Lucene version it uses into an isolated module.

Looking forward I'd hope we wouldn't actually maintain the driver modules - in this case the datastax driver but the rule might apply to all - but have a separate project to build that module. Until then it's better this uses the OGM namespace.

For example the Infinispan integration doesn't ship the Infinispan driver, but depends on the module: much cleaner.